pages/docs/concepts/message.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Message
+weight: 2
+---
+
+## What is a message?
+`Message` is a method of communication used to transmit or exchange information from a sender to the receiver through the use of `channels`. A `message` is generic and can also be defined as an event, command, or request. The sender includes a payload that contains data that would be processed according to the preferred format defined by the receiver.
+
+## Messages vs Events
```

Comment:
from whole twitter conversation about message and events, this is my favorite https://twitter.com/hovenko/status/1569746909186740225 😄 maybe one day we can turn it into a kind of diagram/picture showing Archimedes https://img.freepik.com/darmowe-wektory/archimedes-w-kreskowce-w-wannie-ze-slowem-eureka_1308-100068.jpg?w=2000 as event and then another picture with him where he runs and screams "hey everybody, listen what I've found out" as a message 😆